### PR TITLE
feat: [OeX_Cred-778] update Open Badges v3 verifiable credential composition

### PR DESCRIPTION
**Pull request by wowkalucky**
**Run JavaScript tests locally with Karma**

There is work being done on a fix to get Karma to run in CI. Until then, however, contributors are required to run these tests locally.

- [ ] Make sure you are inside the devstack container
- [ ] Run `make test-karma`
- [ ] All tests pass

### CODE REVIEW

File: credentials/apps/verifiable_credentials/composition/open_badges.py [hunk 1]
```diff
@@ -83,6 +94,8 @@ def get_types(cls):
 class OpenBadges301DataModel(OpenBadgesDataModel):  # pylint: disable=abstract-method
     """
     Open Badges data model.
+
+    NOTE: for example purposes, for now.
```
Comment (GlugovGrGlib) [inline credentials/apps/verifiable_credentials/composition/open_badges.py:98]: Maybe we should be more specific for community, e.g., like this:

```suggestion
    Open Badges version 3.0.1 data model.

    NOTE: this is example of extending data model with custom fields and overrides.
```

**Review by GlugovGrGlib — CHANGES_REQUESTED** [review]
Looks good, the only part that should be changed is with the credentials description template and unused function

File: credentials/apps/verifiable_credentials/issuance/models.py [hunk 2]
```diff
@@ -94,15 +99,90 @@ def issuer_name(self):
...
+        }
+        return capitalize_first(contenttype_to_name.get(self.credential_content_type))
+
+    @property
+    def credential_description(self):
+        """
+        Verifiable credential achievement description resolution.
+        """
+        program_certificate_description = _(
+            "{credential_type} is granted on program {program_title} completion offered by {organizations}, in collaboration with {platform_name}. The {program_title} program includes {course_count} course(s) (, with total {hours_of_effort} Hours of effort required to complete it.)"  # pylint: disable=line-too-long
```
[the remark this answers is not in the document]
Comment (wowkalucky) [reply credentials/apps/verifiable_credentials/issuance/models.py:135]: > Square brackets possibly should be used for the optionality 😸 
No problem.

**Review by GlugovGrGlib — APPROVED** [review]
LGTM!

**Pull request merged**